README.md

```diff
@@ -1,45 +1,54 @@
-# ThermImageAnalysis
-* `CThermal.py` contains classes for sensor value-->temperature conversion from FLIR files (SEQ, images), and the analysis tools for the same, and for splitting SEQ videos
-* `thermal_analysys.py` contains the main code to be run for the analysis tools on the FLIR files.
-* `split_seq.py` splits an SEQ file into multiple FFF files
+# Thermal Image Analysis
+A tool for analyzing and annotating thermal images.
+ - `main.py` contains code for GUI and canvas tools.
+ - `utils.py` contains code for auxiliary functions.
 
+[![Quality check](https://github.com/detecttechnologies/Thermal_Image_Analysis/actions/workflows/qualitycheck.yml/badge.svg)](https://github.com/detecttechnologies/Thermal_Image_Analysis/actions)
```

Comment:
@mukheshpugal  - a few more changes to be made:
The badge isn't showing up, can you check the same? ![image](https://user-images.githubusercontent.com/10851575/126615427-80902abd-abaa-4bd2-bf4c-25e4801ec50b.png)